corehq/apps/app_manager/views/modules.py

```diff
@@ -475,7 +476,11 @@ def _new_report_module(request, domain, app, name, lang):
     module = app.add_module(ReportModule.new_module(name, lang))
     # by default add all reports
     module.report_configs = [
-        ReportAppConfig(report_id=report._id, header={lang: report.title})
+        ReportAppConfig(
+            report_id=report._id,
+            header={lang: report.title},
+            uuid=random_hex(),
```

Comment:
Could/should you set it here instead? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L3306